ucsm_apis/admin/dns.py

```diff
@@ -0,0 +1,149 @@
+# Copyright 2017 Cisco Systems, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+This module performs the operation related to dns server management.
+"""
+from ucsmsdk.ucsexception import UcsOperationError
+from ..common.utils import get_device_profile_dn
+ucsc_base_dn = get_device_profile_dn(name="default")
+
+_dns_server_parent_dn = "sys/svc-ext/dns-svc"
+
+def dns_server_get(handle, name, caller="dns_server_get"):
```

Comment:
Here also we are calling `dns_server_get` in modify, delete and exist API, similar to boot one.